errorhandling.js

```diff
@@ -0,0 +1,20 @@
+exports.handlePsqlErr = (err, req, res, next) => {
+    if (err.code === "22P02") {
```

Comment:
It is ok for you to have them setup but remember that you do not really need your error handling middleware at the moment 🙂